.gitignore

```diff
@@ -33,6 +34,9 @@ venv/
 .vagrant/*
 .lock
 yarn-error.log
+oppia-release-credentials.json
```

Comment:
Hm, pretty sure this shouldn't be here. Did you merge from develop correctly? Please double-check the "Files Changed" tab to ensure that the changes are as expected.

Reply:
I mistakenly added this in a merge. Removed.